Model/Api/Debug.php

```diff
@@ -39,31 +40,44 @@ class Debug implements DebugInterface
 	 */
 	private $pluginVersionFactory;
 
+	/**
+	 * @var ProductMetadataInterface
+	 */
+	private $productMetadata;
+
 	/**
 	 * @param DebugInfoFactory $debugInfoFactory
 	 * @param BoltConfigSettingFactory $boltConfigSettingFactory
 	 * @param PluginVersionFactory $pluginVersionFactory
+	 * @param ProductMetadataInterface $productMetadata
 	 */
 	public function __construct(
 		DebugInfoFactory $debugInfoFactory,
 		BoltConfigSettingFactory $boltConfigSettingFactory,
-		PluginVersionFactory $pluginVersionFactory
+		PluginVersionFactory $pluginVersionFactory,
+		ProductMetadataInterface $productMetadata
 	) {
-		$this->debugInfoFactory         = $debugInfoFactory;
+		$this->debugInfoFactory = $debugInfoFactory;
 		$this->boltConfigSettingFactory = $boltConfigSettingFactory;
-		$this->pluginVersionFactory     = $pluginVersionFactory;
+		$this->pluginVersionFactory = $pluginVersionFactory;
+		$this->productMetadata = $productMetadata;
 	}
 
 	/**
 	 * This request handler will return relevant information for Bolt for debugging purpose.
 	 *
-	 * @api
 	 * @return DebugInfo
+	 * @api
 	 */
-	public function debug() {
+	public function debug()
+	{
 		$result = $this->debugInfoFactory->create();
-		$result->setPhpVersion("todo");
-		$result->setPlatformVersion("todo");
+
+		# populate php version
+		$result->setPhpVersion(phpversion());
```

Comment:
Kind of a nit, but it's probably cleaner to use `PHP_VERSION` as it has no function call overhead.
https://www.php.net/manual/en/reserved.constants.php

Reply:
fixed